distribution/changelog.txt

```diff
@@ -9,6 +9,7 @@
 - Fix: [#11005] Company value overflows.
 - Fix: [#11027] Third color on walls becomes black when saving.
 - Improved: [#11157] Slimmer virtual floor lines.
+- Fix: [#11063] Scrolling position persists when switching tabs in the scenery window.
```

Comment:
The changelog is grouped by change type (fix/feature/improved/etc.). This needs to be moved up one row.